config/locales/exceptions.en.yml

```diff
@@ -116,7 +116,7 @@ en:
         <<: *defaults
         title: Not found
         detail: "There are no routes matching your request: %{path}"
-        code: 411
```

Comment:
@saneshark I know it's been 4 years, but do you remember why this was set as 411 instead of 404?

Reply:
code and http status are distinct -- CODE should be unique to detail.

Reply:
I characterize code as a set of exception codes that VA.gov can raise / return, totally distinct from HTTP Status code. In fact, we have codes of "SM101" etc for that very reason as well with no direct correlation to HTTP Status.

Reply:
Thanks for the clarification.  How/where i the `code` used?

Reply:
exceptions.en.yml file deliberately done with future localization in mind, so that a type of error would have the same code, but you could in theory have a spanish, english, japanese version if you wanted. And of course, any API consumer would then have a means of reliably overriding the VA.gov specific language should they want to as well.